app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrations.jsx

```diff
@@ -56,6 +56,11 @@ export const Integrations = () => {
     setPlugin({});
```

Comment:
And rename this function to smth like `goToInitial` as it set the plugin to initial.